### DISCUSSION TITLE
Help! - Tags transferring un-nested

### DISCUSSION THREAD

**Discussion by qwarz212** [akosbalasko/yarle#406] [General]
Hi everyone,
I'm kinda new at this, but any help would be much appreciated. I have successfully exported all my Evernote notebooks with all attributes checked. But, when I run them through Yarle with the generate nested-tags option, the notes displays as standard, un-nested tags without any parent tag (i.e., as "#Philosophy" and "#Topics", and not "#Topics/Philosophy").
I've played around by adding the EN separator, changing it, tried Windows Yarle and Mac Yarle. Nothin.
Is there an export option I may be missing in EN? Some nested-tag specific code I should include in the template?
Thanks in advance!

**Comment by akosbalasko** [comment]
hi @qwarz212 ,

sorry, i used to totally forget  the discussion forum, and i don't get any notification about the new topics. 
To answer your question: it depends on how you applied the nested tags in Evernote, what is the character which is used to separate the tags?

**Reply by qwarz212** [reply]
I May have accidentally duplicated the issue. Me talked about this in #411
